plugins/Highway_Lanes.py

```diff
@@ -35,7 +35,7 @@ def init(self, logger):
         self.errors[31603] = self.def_class(item = 3160, level = 2, tags = ['highway', 'fix:chair'],
             title = T_('Conflict between usage of *:lanes or *:lanes:(forward|backward|both_ways)'),
             detail = T_(
-'''You can not set at the same time a tag and the variante with this
+'''You can not set at the same time a tag and the variants with this
```

Comment:
Suggestion:
You can not simultaneously set a tag and the variants with `forward`, `backward` or `both_ways` prefixes.

Reply:
(The 6x"`" got lost due to the markup of Github, but I guess you'll know where they're supposed to be :) )

Reply:
Resolved by https://github.com/osm-fr/osmose-backend/pull/1704/commits/60bce7dab2ea21cc02be060ba6dea1060708dcda